kuma/javascript/src/article.jsx

```diff
@@ -55,7 +55,9 @@ export default function Article({ document }: DocumentProps) {
     // This is a one-time effect we need to call the first time an article
     // is rendered, to ensure that interactive examples resize themselves
     // if the browser width changes.
-    useEffect(InteractiveExamples.makeResponsive, []);
+    useEffect(() => {
```

Comment:
This is to fix an eslint error from `react-hooks`, it said to make this an inline function.